README.md

```diff
@@ -30,6 +31,29 @@ The code in this repository is licensed the Apache 2.0 license unless otherwise
 noted. Please see the [LICENSE file](https://github.com/edx/ux-pattern-
 library/blob/master/LICENSE.txt) for details.
 
+## Dependencies
+
+Using the edX Pattern Library source code in a project current requires the following:
+
+* Your project use and can compile Sass into CSS using perferrably [LibSass](http://sass-lang.com/libsass). The edX Pattern Library currently does not provide pre-compiled CSS files.
+* You to locally install [Bower](http://bower.io/) - [see below](#development---ux-pattern-library-package) for more installation details.
+* Your project's Sass manually calls ``edx-pattern-library`` Sass partials in its main compile file.
+* Use of modern web browsers - we support what [Open edX/edX support](http://docstrings.readthedocs.org/en/latest/front_matter/browsers.html)
+
+
+### Third Party Dependencies
+
+Also, the Pattern Library currently relies on the following thrd party libraries:
+
+| Library  | Version |
+| ------------- | ------------- |
+| [Bourbon](https://github.com/thoughtbot/bourbon)  | 4.2.3  |
+| [Animate.css](https://github.com/daneden/animate.css)  | 3.3.0  |
+| [bi-app-sass](https://github.com/anasnakawa/bi-app-sass)  | latest  |
+| [breakpoint-sass](https://github.com/at-import/breakpoint)  | 2.6.1  |
+| [susy](https://github.com/ericam/susy)  | 2.2.5  |
+| [svg4everybody](https://github.com/jonathantneal/svg4everybody)  | 1.0.0  |
```

Comment:
This is a helpful section! 

FWIW - an extra column with a 3-5 word summary per library might be helpful (but far from necessary!) (ex: Sass utilities and mixins [Bourbon], right to left Sass mixins [Bi-app-sass], etc.